studio/local_worker.py

```diff
@@ -25,12 +23,13 @@ class LocalExecutor(object):
     """Runs job while capturing environment and logs results.
     """
 
-    def __init__(self, args):
+    def __init__(self, queue, args):
         self.config = args.config
 
         if args.guest:
             self.config['database']['guest'] = True
 
+        self.task_queue = queue
```

Comment:
We now have to pass tasks queue as a dependency here,
so that we can monitor its state in executor process.